plugins/profiles/profiles.pl

```diff
@@ -72,18 +72,23 @@ sub onStart {
 		$profile = undef;
 	}
 
-	if ( !$profile && @profiles ) {
-		my $choice = $interface->showMenu(	#
-			"Please choose a Profiles folder.",
-			\@profiles,
-			title => "Profiles Selector"
-		);
-
-		my $num = @profiles;
-		return 0 if $choice == $num - 1;
-		return $quit = 1 if $choice == -1;
-
-		$profile = $profiles[$choice];
+	if (!$profile) {
+		if (@profiles > 1) {
+			my $choice = $interface->showMenu(	#
+				"Please choose a Profiles folder.",
+				\@profiles,
+				title => "Profiles Selector"
+			);
+
+			my $num = @profiles;
+			return 0 if $choice == $num - 1;
+			return $quit = 1 if $choice == -1;
+
+			$profile = $profiles[$choice];
+		} else {
+			message "No profiles found, using standard control folder\n";
+			$profile = $profiles[0];
```

Comment:
We shouldn't do this. `@profiles` will be `("Use standard control folder")` here, and we end up adding that to the list of control directories. It won't break anything, because that folder can't exist if we got here, but it won't help either. Just remove this line, and it should all work fine. Better yet, move this logic up and make it an early return instead:
```
if (@profiles == 1) {
    message "No profiles found, using standard control folder\n";
    return;
}
```